scripts/config/pages.js

```diff
@@ -76,6 +76,7 @@ module.exports = [
     ['tour',                                    'static/tour',                                 'full_width', 'Tour'],
     ['why-us',                                  'static/why_us',                               'full_width', 'Why Us'],
     ['platforms',                               'static/platforms',                            'full_width', 'Trading Platforms'],
+    ['dmt5signals',                             'static/dmt5_signals',                         'full_width', 'DMT5 Signals'],
```

Comment:
in binary we call it MT5, not DMT5. DMT5 is our Deriv MT5 platform.

Reply:
Made new pr for this issue https://github.com/binary-com/binary-static/pull/6444